content/docs/1.5.0/references/settings.md

```diff
@@ -98,6 +99,12 @@ Define the policy to use when a node with the last healthy replica of a volume i
 
 Longhorn will generate system snapshot during replica rebuild, and if a user doesn't setup a recurring snapshot schedule, all the system generated snapshots would be left in the replica, and user has to delete them manually, this setting allow Longhorn to automatically cleanup system generated snapshot before and after replica rebuild.
 
+#### Automatically Cleanup Recurring Job Backup Snapshot
+
+> Default: `true`
+
+Longhorn will generate a snapshot during a backup task of a recurring job, and Longhorn will only keep one snapshot that last backup used even though the retain number is not one. This setting allow users to keep the snapshots corresponding to the backups on the remote backup target.
```

Comment:
Suggested wording:
When running a recurring backup job, Longhorn takes a new snapshot before creating the backup. Longhorn retains only the snapshot used by the last backup job even if the value of the retain parameter is not 1. This setting ensures that the retained snapshots directly correspond to the backups on the remote backup target.

Reply:
Moved to 1.6.0 and Modified. Thanks for suggestions.